### DISCUSSION TITLE
Homebrew not installing getting no such file or directory error

### DISCUSSION THREAD

**Discussion by bripark3** [Homebrew/discussions#2054] [Getting started] [answered]
I’m unable to install homebrew on my iMac. I’m running 11.5.2 Big Sur. I am copying and pasting the code straight from the homebrew page and receiving this error. I meet all the requirements so I’m lost as to what my issue may be. 
![574B2926-B5C5-4D15-8DDD-FDAAA0668C70](https://user-images.githubusercontent.com/89730913/131281094-e819ec4b-81f9-4339-8e85-1863d2a4b3ed.jpeg)

**Comment by carlocab** [comment, marked as answer]
Your shell is inserting extra `\`s when you paste the command. Try removing those first.

**Reply by bripark3** [reply]
Thank you so much this was exactly what the issue was.
